versioned_docs/version-5.1/general-concepts/multilingual.md

```diff
@@ -1,3 +1,202 @@
+
+### Language Files in Components
+
+:::info
+Since Joomla Version 4 and 5, you do not need the language prefix in the language file name.
+:::
+
+```plaintext
+components
+    com_example
+        language
+            en-GB
+                com_example.ini
+                com_example.sys.ini
+```
+
+### Language Files in Modules
+
+```plaintext
+modules
+    mod_example
+        language
+            en-GB
+                mod_example.ini
+                mod_example.sys.ini
+```
+
+### Including Language Files in the Manifest
+
+To ensure your extension's language files are recognized, include them in the **extension's manifest file**.
+
+```xml
+<languages>
+    <language tag="en-GB">language/en-GB/mod_example.ini</language>
+    <language tag="en-GB">language/en-GB/mod_example.sys.ini</language>
+</languages>
+```
+
+For more details on the manifest file, refer to the [Joomla documentation](https://manual.joomla.org/docs/building-extensions/modules/module-development-tutorial/step4-languages/).
+
+### Database Schema
+
+For components, ensure to include a 'language' column in your database tables to store the language code of the content. 
+
+Doing so allows you to filter content based on the language.
+
+```sql
+CREATE TABLE `#__example` (
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255) NOT NULL,
+    `language` CHAR(7) NOT NULL,
+    PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+```
+
+This column should be populated with the language code (e.g., `en-GB`, `fr-FR`) when saving content.
+
+## Debugging Multilingual Issues
+
+Joomla offers robust debugging tools to help identify untranslated strings and resolve issues with language translations in your installed extensions. 
+
+Here's how to enable and use these tools:
+
+1. **Access Global Configuration:**
+    - From the Home Dashboard, click on the **Global Configuration** button in the System panel.
+
+2. **Enable Debug Language:**
+    - In the System panel, toggle **Debug Language** to **Yes**.
+
+3. **Adjust Language Display:**
+    - The default setting for Language Display is **Value**. Changing it to **Constant** may disrupt the layout due to long constants not wrapping properly.
+
+When **Debug Language** is activated, Joomla highlights all translatable values with special characters that indicate their status:
+
+- **Translated Strings:**
+    - Text surrounded by asterisks (\**Joomla CMS\**) shows that the string has been successfully translated from a language file.
+
+- **Untranslated Strings:**
+    - Text surrounded by question marks (??Joomla CMS??) signifies that the string is intended for translation, but no corresponding entry was found in the language files.
+
+- **Non-Translatable Strings:**
+    - Text without any special characters (Joomla CMS) indicates that the string is not set up for translation.
+
+Utilizing these debugging features will help you quickly identify and fix translation issues in your Joomla extensions.
+
```

Comment:
You should also include information from the output of the debug plugin eg
![image](https://github.com/user-attachments/assets/b7909bc6-c95a-4851-8528-9c58520b15c0)

Reply:
May I use your screenshot?

Reply:
Added it